src/components/ContentSidebar/__tests__/SidebarFileProperties-test.js

```diff
@@ -42,15 +42,30 @@ describe('components/ContentSidebar/SidebarFileProperties', () => {
             defaultMessage: 'baz'
         };
 
-        const props = {
+        const errorProps = {
             inlineError: {
                 title: fakeError,
                 content: fakeError
             }
         };
-        const wrapper = shallow(<SidebarFileProperties {...props} />).dive();
+        const wrapper = shallow(<SidebarFileProperties {...errorProps} />).dive();
 
         expect(wrapper.find(InlineError)).toHaveLength(1);
         expect(wrapper).toMatchSnapshot();
     });
+
+    test('should render classification information', () => {
+        props.hasClassification = true;
```

Comment:
It doesnt seem like this test does much, looking at the snapshot. I would try using mount, and using enzyme to assert the classification component renders

Reply:
Updated, let me know if that's what you were thinking.

Reply:
Yea I think thats good. Might make sense to have the inverse test as well